UM/i18n.py

```diff
@@ -30,12 +30,15 @@ class i18nCatalog: # [CodeStyle: Ultimaker code style requires classes to start
     #   \param language The language to load. Valid values are language codes or
     #   "default". When "default" is specified, the language to load will be
     #   determined based on the system"s language settings.
+    #   \param fallback_names The fallback names of the catalog to load. If a translation is not found in the primary
+    #   name, it will try to find it in the next fallback.
```

Comment:
Can you explain to me again what the problem was that necessitates this?

We are here adding a structure to the i18n catalogue that when reading the i18n catalogue code makes no sense as to why it is necessary to have fallback names. We only have one translation active at a time, so why would a translation not be found and where would you look otherwise?

Reply:
Explained in the JIRA ticket